scripts/tools/rl_kube_warmup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Usage:
+# bash rl_worker_kube_warmup.sh [FLAGS]
```

Comment:
@vkartik97 Can you add an example usage with the flags? 
Also, give a short description of what this script does.